### ISSUE TITLE
Follow-up question on heading punctuation

### ISSUE THREAD

**Issue by akirscht** [MicrosoftDocs/microsoft-style-guide#45]
Can you provide guidance on whether the "Don't end headings with a period" applies in situations such as

Build. Sell. Grow.

or

Not seeing what you want? We can help.

It would seem weird/inconsistent to leave off the end period when punctuation was used before, no?  

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1598da0f-c10d-05fa-a5c3-6e52bc511c35
* Version Independent ID: 353b7f63-2c35-667b-f898-e55ad21558ce
* Content: [Headings - Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/scannable-content/headings)
* Content Source: [styleguide/scannable-content/headings.md](https://github.com/MicrosoftDocs/microsoft-style-guide/blob/master/styleguide/scannable-content/headings.md)
* Product: **non-product-specific**
* GitHub Login: @pallep
* Microsoft Alias: **pallep**

**Comment by camind** [comment]
Hi,

The guideline not to end headings with a period is particularly applicable to *headings*—the sentence fragments that break up long blocks of text into a hierarchical structure, like Heading 1, Heading 2, and so on.
 
I think it’s OK to occasionally have a *headline* that consists of several short sentences (like your example). This kind of thing usually occurs in marketing content, such as the marketing pages on websites. In these designs, the large text is more like body text set in a large font than it is like a heading that provides hierarchical structure.

In other words, they’re short paragraphs, they’re just in big type. So the periods make perfect sense.

I hope that helps!
Catherine
